visualization/scripts/convert_forecasts.py

```diff
@@ -53,7 +53,8 @@ def reformat_forecasts(file_path, target, root, data):
               'Montana', 'Nebraska', 'Nevada', 'New Hampshire', 'New Jersey', 'New Mexico', 'New York',
               'North Carolina', 'North Dakota', 'Ohio', 'Oklahoma', 'Oregon', 'Pennsylvania', 'Rhode Island',
               'South Carolina', 'South Dakota', 'Tennessee', 'Texas', 'Utah', 'Vermont', 'Virginia', 'Washington',
-              'West Virginia', 'Wisconsin', 'Wyoming', 'District of Columbia']
+              'West Virginia', 'Wisconsin', 'Wyoming', 'District of Columbia',
+              'Puerto Rico', 'Virgin Islands', 'Guam', 'Northern Mariana Islands', 'American Samoa']
```

Comment:
Why would we need these right now if we cannot visualize them yet? I see that we have not added these to the map as of now, right?

Reply:
No, not yet. But they can be selected through the drop-down menu, which is useful. The way that the map visualization works makes it hard to find a map with US state boundaries and all the territories, so this is a temporary solution while I try to find a working map.